composer.json

```diff
@@ -24,6 +24,7 @@
     },
     "require-dev": {
         "dbrekelmans/bdi": "^1.0",
+        "justinrainbow/json-schema": "^5.2",
```

Comment:
Maybe add it to the "suggests" section too.

Reply:
yep, you're right!
it's done.